src/actions/requests.rs

```diff
@@ -321,7 +323,7 @@ impl RequestAction for Completion {
         let vfs = ctx.vfs;
         let file_path = parse_file_path!(&params.text_document.uri, "complete")?;
 
-        let cache = racer::FileCache::new(vfs);
+        let cache = racer_cache(vfs);
```

Comment:
I realize this is minor and a thing of preference, but personally I'd find it a lot easier to read with `racer::FileCache::new(RacerVfs(vfs))` as this limits what I'd expect to need to know - here it'd just be a `impl racer::FileLoader for RacerVfs/*(Arc<Vfs>)*/` that is hiding somewhere 😉 

Super minor though!